### DISCUSSION TITLE
Using the MonAi procedure causes the following error

### DISCUSSION THREAD

**Discussion by StayNightShip** [Project-MONAI/MONAI#7242] [Q&A]
ValueError: Unsupported spatial_border length: 6, available options are [1, len(spatial_shape)=1, 2*len(spatial_shape)=2].

**Comment by KumoLiu** [comment]
Hi @StayNightShip, could you please paste more information such as what transform you use and what's your input? 
From the error message, seems the `spatial_border` you specified exceeds the length.

Thanks!
